readthedocsext/theme/templates/projects/partials/edit/temporary_access_list.html

```diff
@@ -0,0 +1,108 @@
+{% extends "includes/crud/table_list.html" %}
+
+{% load i18n %}
+
+{% load core_tags %}
+{% load privacy_tags %}
+{% load projects_tags %}
+
+{% block top_left_menu_items %}
+{% endblock top_left_menu_items %}
+
+{% block create_button %}
+  <div class="item">
+    <a class="ui green icon button"
+       href="{% url "projects_temporary_access_create" project.slug %}">
+      <i class="fa-solid fa-plus icon"></i>
+      {% trans "Add share" %}
+    </a>
+  </div>
+{% endblock %}
+
+{% block list_placeholder_icon_class %}fa-duotone fa-share-nodes{% endblock %}
+{% block list_placeholder_header %}
+  {% blocktrans trimmed %}
+    No active shares are configured yet
+  {% endblocktrans %}
+  <div class="sub header">
+    {% blocktrans trimmed %}
+      Sharing tokens can be used to share private documentation projects to users outside your organization.
+    {% endblocktrans %}
+  </div>
+{% endblock list_placeholder_header %}
+{% block list_placeholder_text %}
+  <a class="ui button" href="https://docs.readthedocs.io/page/commercial/sharing.html">{% trans "Learn more" %}</a>.
+{% endblock list_placeholder_text %}
+
+{% block list_item_right_buttons %}
+  {% if object.access_type != "http_header_token" %}
+    {# TODO make this a dropdown menu so we can provide a bit more context #}
+    {# TODO maybe also make this button disabled if there is not sharing URL? #}
+    <a class="ui icon button"
+       href="{{ object.get_absolute_url }}"
+       data-content="Share access URL">
+      <i class="fa-solid fa-share-nodes icon"></i>
+    </a>
+  {% endif %}
+
+  {% url "projects_temporary_access_delete" object.project.slug object.pk as form_url %}
+  {% trans "Remove share" as action_text %}
+  {% blocktrans trimmed asvar content_text with share=object.description %}
+    Remove share {{ share }}?
+  {% endblocktrans %}
+  {% include "includes/crud/remove_button.html" with id=object.pk form_url=form_url action_text=action_text context_text=context_text %}
+{% endblock list_item_right_buttons %}
+
+{% block list_item_icon %}
+  {% if object.access_type == "password" %}
+    <i class="fa-duotone fa-input-text icon" data-content="{{ object.get_access_type_display }}"></i>
+  {% elif object.access_type == "token" %}
+    <i class="fa-solid fa-link icon" data-content="{{ object.get_access_type_display }}"></i>
+  {% elif object.access_type == "http_header_token" %}
+    <i class="fa-solid fa-right-left-large icon" data-content="{{ object.get_access_type_display }}"></i>
+  {% endif %}
+{% endblock list_item_icon %}
+
+{% block list_item_header %}
+  <a href="{% url "projects_temporary_access_edit" project.slug object.pk %}">
+    {{ object.description|truncatechars:24 }}
+  </a>
+  <div class="sub header">
+    {% if object.allow_all %}
+      {% blocktrans trimmed %}
+        All versions
+      {% endblocktrans %}
+    {% else %}
+      {# TODO consider listing the actual versions here #}
+      {% blocktrans trimmed %}
+        Select versions
```

Comment:
```suggestion
        Selected versions
```

"Select versions" feels like an action not sure, it's probably right :thinking:

Reply:
Yeah, honestly neither are a really good fit, I'd rather replace this with something listing the versions explicitly, or at least counting them ("5 versions"). I would use "select" over "selected", as the past tense version is more ambiguous about what we previously selected.

I was going to save this, but if I have to jump back in, maybe I'll just replace this text now. I think a simple count works fine.